core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1322,33 +1322,55 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
 
   test("SPARK-20640: Shuffle registration timeout and maxAttempts conf are working") {
     val tryAgainMsg = "test_spark_20640_try_again"
+    val `timingoutExecutor` = "timingoutExecutor"
```

Comment:
Why the backticks?

Reply:
No real reason, just a mistake made when renamed the value.